### PR TITLE
Parse symbol table and symbol code

### PR DESCRIPTION
**Pull request by Meisterschueler**

### CODE REVIEW

File: src/position.rs [hunk 1]
```diff
@@ -9,6 +9,8 @@ pub struct AprsPosition {
     pub timestamp: Option<Timestamp>,
     pub latitude: f32,
     pub longitude: f32,
+    pub symbol_table: char,
+    pub symbol_code: char,
```
Comment (Turbo87) [inline src/position.rs]: I'm wondering if it would make sense to use an `enum` for these 🤔

Comment (Meisterschueler) [reply src/position.rs]: I tried out...

**Comment by Meisterschueler** [comment]
Finally I managed it... @Turbo87 : is this the correct direction?

**Comment by Turbo87** [comment]
@Meisterschueler sorry, been pretty busy the past few weeks and somehow forgot about this. I'll try to review as soon as I can :-/

**Comment by Meisterschueler** [comment]
No problem. In the meantime I think that it is a bad idea to get an enum from symbol_table and symbol_code. The meaning of symbol_table+symbol_code is really really really poor specified. We should keep outside the poor specified area.

**Pull request closed without merging**